### PR TITLE
Prometheus documentation

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?
Adding Prometheus check documentation

### Motivation
Some Agent checks are now based on Prometheus, this doc explains how to use the `PrometheusCheck` interface

### Additional Notes
https://github.com/DataDog/dd-agent/pull/3317#issuecomment-339392893

### CODE REVIEW

File: content/agent/prometheus.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
+---
+title: Prometheus Check
+kind: documentation
+---
+
+## Overview
+
+This page details how to collect metrics and events from a Prometheus data source by writing an Prometheus Check, a Python plug-in to the Datadog Agent. This page looks first at the `PrometheusCheck` interface, and then propose a simple Prometheus Check that collects timing metrics and status events from [Kube DNS](https://github.com/DataDog/integrations-core/blob/master/kube_dns/check.py).
```
Comment (jeremy-lq) [inline content/agent/prometheus.md]: nit s/an Prometheus Check/a Prometheus Check/
I'd remove "a Python plug-in to the Datadog Agent", as it results in an awkward sentence. All custom check are a Python plug-in to the Datadog Agent.

File: content/agent/prometheus.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+## Setup
```
Comment (jeremy-lq) [inline content/agent/prometheus.md]: I'd remove this section.

File: content/agent/prometheus.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+First off, ensure you've properly installed the [Agent](http://app.datadoghq.com/account/settings#agent) on your machine. If you run into any issues during the setup, [contact our support](/help).
+
+## Prometheus check interface 
+
+`PrometheusCheck` is [a mother class](https://github.com/DataDog/dd-agent/blob/master/checks/prometheus_check.py) providing a structure and some helpers
```
Comment (jeremy-lq) [inline content/agent/prometheus.md]: `PrometheusCheck` is [a mother class](https://github.com/DataDog/dd-agent/blob/master/checks/prometheus_check.py) providing a structure and some helpers to collect metrics, events, and service checks exposed via Prometheus. Minimal configuration for checks based on this class include:

File: content/agent/prometheus.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+
+class KubeDNSCheck(PrometheusCheck):
+    def __init__(self, name, init_config, agentConfig, instances=None):
+        super(KubeDNSCheck, self).__init__(name, init_config, agentConfig, instances)
+        self.NAMESPACE = 'kubedns'
+```
+
+#### Overriding `self.metrics_mapper`
+
+`metrics_mapper` is a dictionary where the keys are the metrics to capture and the values are the corresponding metrics names to have in datadog.
```
Comment (jeremy-lq) [inline content/agent/prometheus.md]: `metrics_mapper` is a dictionary where the key is the metric to capture and the value is the corresponding metric name in Datadog.

File: content/agent/prometheus.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+We override it so all metrics reported by your Prometheus checks are not counted as [custom metric](/getting_started/custom_metrics):
```
Comment (jeremy-lq) [inline content/agent/prometheus.md]: The reason for the override it so metrics reported by the Prometheus checks are not counted as [custom metric](/getting_started/custom_metrics):

File: content/agent/prometheus.md (hunk 1 above)
Comment (l0k0ms) [reply content/agent/prometheus.md]: 👍

File: content/agent/prometheus.md (hunk 2 above)
Comment (l0k0ms) [reply content/agent/prometheus.md]: 👍

File: content/agent/prometheus.md (hunk 3 above)
Comment (l0k0ms) [reply content/agent/prometheus.md]: 👍

File: content/agent/prometheus.md (hunk 4 above)
Comment (l0k0ms) [reply content/agent/prometheus.md]: 👍

File: content/agent/prometheus.md (hunk 5 above)
Comment (l0k0ms) [reply content/agent/prometheus.md]: 👍

File: content/agent/agent_checks.md [hunk 6]
```diff
@@ -1,13 +1,15 @@
...
+customnav: agentnav
 aliases:
   - /guides/agent_checks/
 ---
 
 ## Overview
 
-This page details how to collect metrics and events from a new data source by writing an Agent Check, a Python plug-in to the Datadog Agent. This page looks first at the `AgentCheck` interface, and then propose a simple Agent Check that collects timing metrics and status events from HTTP services.
+This page details how to collect metrics and events from a new data source by writing an Agent Check.  
+This page looks first at the `AgentCheck` interface, and then propose a simple Agent Check that collects timing metrics and status events from HTTP services.
```
Comment (jeremy-lq) [inline content/agent/agent_checks.md]: This page first looks at the `AgentCheck` interface, and then proposes a simple Agent Check that collects timing metrics and status events from HTTP services.

File: content/agent/prometheus.md [hunk 7]
```diff
@@ -0,0 +1,201 @@
+---
+title: Prometheus Check
+kind: documentation
+---
+
+## Overview
+
+This page details how to collect metrics and events from a Prometheus data source by writing a Prometheus Check.  
+This page looks first at the `PrometheusCheck` interface, and then propose a simple Prometheus Check that collects timing metrics and status events from [Kube DNS](https://github.com/DataDog/integrations-core/blob/master/kube_dns/check.py).
```
Comment (jeremy-lq) [inline content/agent/prometheus.md]: This page first looks at the `PrometheusCheck` interface, and then proposes a simple Prometheus Check that collects timing metrics and status events from [Kube DNS](https://github.com/DataDog/integrations-core/blob/master/kube_dns/check.py).

File: content/agent/agent_checks.md (hunk 6 above)
Comment (l0k0ms) [reply content/agent/agent_checks.md]: 👍

**Pull request merged**